contracts/eos/Common/common.hpp

```diff
@@ -78,41 +91,95 @@ std::string build_memo(memo_structure data) {
  *  e.g. - to_fixed(14.214212, 3) --> 14.214
 */
 double to_fixed(double num, int precision) {
-    return (int)(num * pow(10, precision)) / pow(10, precision);
+    return (int)(num * pow(10.0, precision)) / pow(10.0, precision);
+}
+
+float stof(const char* s) {
+    float rez = 0, fact = 1;
+    
+    if (*s == '-') s++; //skip the sign
```

Comment:
we fixed it on master, we need to support negative numbers here. small change that can prevent issues in the future

Reply:
Yeah, will update